vendor/engines/secure_rooms/app/controllers/secure_rooms/card_readers_controller.rb

```diff
@@ -0,0 +1,63 @@
+module SecureRooms
+
+  class CardReadersController < ApplicationController
+
+    admin_tab :all
+    customer_tab :password
```

Comment:
yeah haha I don't really know what these mean so some might sneak in along the way